.idea/vehicle-damage-modeling.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
```

Comment:
What is that?

Reply:
I was doing something on Android and I forgot remove .idea folder. This file should be removed. My mistake :/

Reply:
removed